ghost/core/test/unit/frontend/helpers/foreach.test.js

```diff
@@ -282,6 +290,14 @@ describe('{{#foreach}} helper', function () {
             ]
         };
 
+        const newsletterArrayHashWithVisibility = {
```

Comment:
How about we co-locate `newsletterObjectHashWithVisibility` and `newsletterArrayHashWithVisibility` with the test seeming though its unlikely they will be used in other tests?